framework/grid/ActionColumn.php

```diff
@@ -139,7 +150,7 @@ protected function initDefaultButtons()
                     'title' => Yii::t('yii', 'View'),
                     'aria-label' => Yii::t('yii', 'View'),
                     'data-pjax' => '0',
-                ], $this->buttonOptions);
+                ], isset($this->buttonOptions['view']) ? $this->buttonOptions['view'] : $this->buttonOptions);
```

Comment:
@SamMousa Thanks for the note.

I realized it, what do you think if I removed default options?

```
$options = [
     'title' => Yii::t('yii', 'View'),
     'class' => 'btn btn-info btn-xs btn-view',
     'aria-label' => Yii::t('yii', 'View'),
     'data-pjax' => '0',
];
                
if (isset($this->buttonOptions['view'])) {
     $options = array_merge($options, $this->buttonOptions['view']);
}
```